root/static/scripts/global.js

```diff
@@ -1,5 +1,9 @@
+let defaultExport;
+
 if (typeof global === 'undefined') {
-  module.exports = window;
+  defaultExport = window;
 } else {
-  module.exports = global;
+  defaultExport = global;
 }
+
+export default defaultExport;
```

Comment:
There's an import in root/static/scripts/tests/react-macros.js which needs `.default` added

Reply:
Done! (BTW, eslint thinks most variables in that file are unused - should we remove them, should we disable that eslint check on this file, or should we do some third thing?)

Reply:
Ah, we should disable the eslint check on that file (the imports are used in `eval`'d code sent from t/lib/t/TemplateMacros.pm, basically).